app/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
                 }
                 assets {
                     srcDirs += files("$projectDir/../node_modules/@duckduckgo/privacy-dashboard/build/app".toString())
+                    srcDirs += files("$projectDir/../node_modules/@duckduckgo/content-scope-scripts/build/android/pages".toString())
```

Comment:
I have opinions about having the pages under c-s-s. I think we can leave it here for now and add the folder to .gitignore but I'd create a task to move this to the duck player module using `copy-files-from-to` like in reference tests.

Reply:
Leaving this for later